docs/content/how-to/openstack/index.md

```diff
@@ -21,8 +21,10 @@ Table of content:
 
 - [Prerequisites](prerequisites.md)
 - [Create a Hosted Cluster and a default Nodepool](hostedcluster.md)
-- Advanced scenarios: additional Nodepools can be created with specific configurations:
-  - [Nodepools distributed over Nova availability zones](az.md)
-  - [Additional ports for Nodepools](additional-ports.md)
-  - [Node tuning, SR-IOV Network Operator](performance-tuning.md)
 - [Destroy a Hosted Cluster](destroy.md)
+
+Advanced scenarios: additional Nodepools can be created with specific configurations:
```

Comment:
Is the `-` missing here?

Reply:
I think it's fine. I wish I could see it deployed like we used to be able to from the PR.

I checked it out locally and it looks fine.